test/integration/connect/envoy/Dockerfile-fortio-windows

```
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/windows/nanoserver:1809
+
+ENV FORTIO_URL=https://github.com/fortio/fortio/releases/download/v1.33.0/fortio_win_1.33.0.zip
+
+RUN curl %FORTIO_URL% -L -o fortio.zip
+RUN tar -xf fortio.zip
+
+CMD [ "fortio.exe", "server" ]
```

This suggestion will create fortio directory and extract zip flle contents within, Also we add the fortio dir to env path.

```suggestion
FROM mcr.microsoft.com/windows/nanoserver:1809

RUN mkdir fortio

ENV FORTIO_URL=https://github.com/fortio/fortio/releases/download/v1.33.0/fortio_win_1.33.0.zip
RUN curl %FORTIO_URL% -L -o fortio.zip

RUN tar -xf fortio.zip -C fortio

ENV PATH C:\\fortio;%PATH%

CMD [ "fortio.exe", "server" ]
```

---

I think it is an additional step that adds complexity to the recipe since the only purpose is to run the Fortio server. Let me know if we need to change it as you suggest